src/rustllvm/RustWrapper.cpp

```diff
@@ -395,22 +395,25 @@ static InlineAsm::AsmDialect fromRust(LLVMRustAsmDialect Dialect) {
   }
 }
 
-extern "C" LLVMValueRef LLVMRustInlineAsm(LLVMTypeRef Ty, char *AsmString,
-                                          char *Constraints,
-                                          LLVMBool HasSideEffects,
-                                          LLVMBool IsAlignStack,
-                                          LLVMRustAsmDialect Dialect) {
-  return wrap(InlineAsm::get(unwrap<FunctionType>(Ty), AsmString, Constraints,
+extern "C" LLVMValueRef LLVMRustInlineAsm(
+    LLVMTypeRef Ty, char *AsmString, size_t AsmStringLen,
+    char *Constraints, size_t ConstraintsLen,
+    LLVMBool HasSideEffects, LLVMBool IsAlignStack, LLVMRustAsmDialect Dialect) {
```

Comment:
Do we have any conventions for formatting C++ code?
Like "run `clang-format` with LLVM code style".
This PR changes formatting in quite a few places.

Reply:
Git history reveals that clang-format with LLVM style had been used from time
to time to format files in rustllvm. I formatted the commit accordingly.